src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderService.java

```diff
@@ -1168,8 +1168,7 @@ public DraftOrder getCurrentCreateDraftOrderDetails(CaseData caseData, String lo
                             ? copyPropertiesToSdoDetails(caseData) : null)
             .hearingsType(caseData.getManageOrders().getHearingsType())
             .isOrderCreatedBySolicitor(UserRoles.SOLICITOR.name().equals(loggedInUserType) ? Yes : No)
-            .judgeNotes(null != caseData.getManageOrders().getJudgeDirectionsToAdminAmendOrder()
-                            ? caseData.getManageOrders().getJudgeDirectionsToAdminAmendOrder() : " ")
+            .judgeNotes(caseData.getJudgeDirectionsToAdmin())
```

Comment:
Do we need this change? I mean removing this null check and " " was it intentional? If yes, please ignore